ui/src/submissions/conferences/containers/__tests__/ConferenceSubmissionSuccessPageContainer.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { fromJS } from 'immutable';
+
+import ConferenceSubmissionSuccessPageContainer, {
+  ConferenceSubmissionSucessPage,
+} from '../ConferenceSubmissionSuccessPageContainer';
+import { getStoreWithState } from '../../../../fixtures/store';
+
+describe('ConferenceSubmissionSuccessPageContainer', () => {
+  it('passes props to ConferenceSubmissionSucessPage', () => {
+    const store = getStoreWithState({
+      submissions: fromJS({
+        successData: {
+          pid_value: 12345,
+          cnum: 'C2019-02-01',
```

Comment:
This is not a real CNUM, it should be `C19-02-01`

Reply:
well those don't matter, this just testing whatever is here, is passed to the component.